src/views/dialog.rs

```diff
@@ -244,6 +249,11 @@ impl Dialog {
         Box::new(self.buttons.iter_mut().map(|b| &mut b.button.view))
     }
 
+    /// Returns currently focused element
+    pub fn get_current_focus(&self) -> &Focus {
```

Comment:
Let's rename that to simply `focus(&self)` - there can be no other (non-current) focus, and most getters don't need a `get_` prefix.